### PR TITLE
chore(ColorPicker): deprecate enums

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/80492480/pulses/7021048627

### CODE REVIEW

File: packages/core/src/components/ColorPicker/ColorPicker.tsx [hunk 1]
```diff
@@ -51,44 +53,42 @@ export interface ColorPickerProps extends VibeComponentProps {
 }
 
 const ColorPicker: VibeComponent<ColorPickerProps> & {
-  // Backward compatibility for enum naming
-  COLOR_STYLES?: typeof ColorStyle;
```
Comment (talkor) [inline packages/core/src/components/ColorPicker/ColorPicker.tsx:55]: Changelog :)

File: packages/core/src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx [hunk 2]
```diff
@@ -50,19 +52,19 @@ export interface ColorPickerContentProps extends VibeComponentProps {
 
 const ColorPickerContent: VibeComponent<ColorPickerContentProps, HTMLDivElement> & {
   // Backward compatibility for enum naming
-  COLOR_STYLES?: typeof ColorStyle;
+  COLOR_STYLES?: typeof ColorStyleEnum;
```
Comment (talkor) [inline packages/core/src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx:55]: Let's remove and add to changelog?

File: packages/core/src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx [hunk 3]
```diff
@@ -154,9 +156,9 @@ const ColorPickerContent: VibeComponent<ColorPickerContentProps, HTMLDivElement>
 
 export default withStaticProps(ColorPickerContent, {
   // Backward compatibility for enum naming
-  COLOR_STYLES: ColorStyle,
+  COLOR_STYLES: ColorStyleEnum,
```
Comment (talkor) [inline packages/core/src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx:159]: Same

File: packages/core/src/utils/colors-utils.ts [hunk 4]
```diff
@@ -1,12 +1,11 @@
-import { ColorStyle, contentColors } from "./colors-vars-map";
+import { ColorStyle as ColorStyleEnum, contentColors } from "./colors-vars-map";
+import { ColorStyle } from "../types/Colors";
 
 const ColorUtils = {
-  modes: ColorStyle,
+  modes: ColorStyleEnum,
```
Comment (talkor) [inline packages/core/src/utils/colors-utils.ts:5]: Can't we use the type here?

File: packages/core/src/components/ColorPicker/ColorPicker.tsx (hunk 1 above)
Comment (rivka-ungar) [reply packages/core/src/components/ColorPicker/ColorPicker.tsx:55]: Added

File: packages/core/src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx (hunk 2 above)
Comment (rivka-ungar) [reply packages/core/src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx]: Done

File: packages/core/src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx (hunk 3 above)
Comment (rivka-ungar) [reply packages/core/src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx]: Done

File: packages/core/src/utils/colors-utils.ts (hunk 4 above)
Comment (rivka-ungar) [reply packages/core/src/utils/colors-utils.ts:5]: It expects a value not a type

File: packages/core/docs/vibe-3-changelog.md [hunk 5]
```diff
@@ -193,6 +193,16 @@ codemod: `avatar-component-migration`
 - `arrowPosition` removed [codemod]
 - `TooltipArrowPosition` removed [codemod]
 
+### ColorPicker
+
+- `COLOR_STYLES` removed [codemod]
```
Comment (talkor) [inline packages/core/docs/vibe-3-changelog.md:198]: It's a bit unclear IMO, maybe `ColorPicker.COLOR_STYLES`?

**Pull request merged**